source/factory.js

```diff
@@ -16,11 +14,11 @@
  * @param {Object} value
  * @returns {Function}
  */
-module.exports = function (value) {
-    var listeners = [];
+export default value => {
```

Comment:
Can we always wrap the parameter into brackets in such arrow functions (here and in other places)? I.e. `(value) =>` instead of `value =>`. I believe it could increase readability (in fact, it does for me, and most likely there’re other people like me that will also benefit from this).

Also, is there an ESLint rule to enforce such wrapping?